lib/blocks.php

```diff
@@ -45,3 +45,51 @@ function gutenberg_reregister_core_block_types() {
 	}
 }
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
+
+/**
+ * Serializes an array of blocks.
+ *
+ * @since 5.7.0
+ *
+ * @param array $blocks Post Array of block objects.
+ * @return string String representing the blocks.
+ */
+function serialize_blocks( $blocks ) {
```

Comment:
I think these serialize functions shouldn't be "official" functions we support, As far as I understand, they are only needed for the legacy block behavior. They can be local to that or at least mark them as unstable.